roles/install_local_settings/templates/geniza_settings.py

```diff
@@ -12,17 +12,65 @@
     'libraries': '{{ libraries_csv_url }}',
     'languages': '{{ languages_csv_url }}',
     'metadata': '{{ metadata_csv_url }}'
+    'demerged': '{{ demerge_csv_url }}'
 }
-{% endblock %}
 
+# path to preliminary JSON transcription data
+TRANSCRIPTIONS_JSON_FILE = "/srv/www/geniza/data/transcriptions.json"
+
+{% endblock %}
 
 {% block solr_config %}
 from geniza.settings.components.base import SOLR_CONNECTIONS
 
-SOLR_CONNECTIONS['default'].update({
-    'URL': '{{ solr_url }}',
-    'COLLECTION': '{{ solr_collection }}',
-    'CONFIGSET': '{{ solr_configset }}'
-})
+SOLR_CONNECTIONS['default']['URL'] = "{{ solr_url }}"
+SOLR_CONNECTIONS['default']['COLLECTION'] = "{{ solr_collection }}"
+SOLR_CONNECTIONS['default']['CONFIGSET'] = "{{ solr_configset }}"
+
+{% endblock %}
+
 
-{% endblock %}
\ No newline at end of file
+{% block logging %}
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'basic': {
+            'format': '[%(asctime)s] %(levelname)s:%(name)s::%(message)s',
+            'datefmt': '%d/%b/%Y %H:%M:%S',
+        },
+    },
+    'handlers': {
+        'mail_admins': {
+            'level': 'ERROR',
+            'class': 'django.utils.log.AdminEmailHandler',
+            'include_html': True
+        },
+        {# This configuration lets logrotate and its proper permissions #}
+        {# handle this problem #}
+        'debug_log': {
+            'level': 'DEBUG',
+            'class': 'logging.handlers.WatchedFileHandler',
+            'filename': '{{ logging_path }}',
+            'formatter': 'basic',
+        }
+    },
+    'loggers': {
+        'django.request': {
+            'handlers': ['mail_admins', 'debug_log'],
+            'level': 'ERROR',
+            'propagate': True,
+        },
+        'geniza': {
+            'handlers': ['debug_log'],
+            'level': 'WARN',
```

Comment:
We've put some of the status info from `import_data` under "INFO". Will this hide that info? Should we raise the level of more important logs?

Reply:
The script output isn't using this logger, it's configuring and using it's own logger based on verbosity level specified when running the script (I think this is a bit confusing, fwiw).

I think warn should be a reasonable default for qa, and we can always temporarily adjust it if we need to debug something.